docs/DeveloperGuide.md

```diff
@@ -308,6 +308,21 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Finding Clients**
+
+**MSS**
+
+1.  User requests to find a client
+2.  TBM shows a list of clients that match user's query
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list of clients is empty.
```

Comment:
```suggestion
* 1a. The list of clients is empty.
```

Reply:
The extension is specific to step 2 of the MSS ("TBM shows a list of clients that match user's query"), hence the extension should be named `2a` instead of `1a` isn't it?